From af2f24d46d434782aad64397ce90c817e9e4a708 Mon Sep 17 00:00:00 2001
From: Rohit kumar <rohitkr@codeaurora.org>
Date: Tue, 25 Jun 2019 12:52:31 +0530
Subject: [PATCH 3/5] Add support for event handling

Add below support:
1) Event handling
2) Remove passing snd_card_def pointer to plugin

Signed-off-by: Rohit kumar <rohitkr@codeaurora.org>
---
 git/include/tinyalsa/mixer.h        |   2 +
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/git/include/tinyalsa/mixer.h b/git/include/tinyalsa/mixer.h
index faeb957..730ea3f 100644
--- a/git/include/tinyalsa/mixer.h
+++ b/git/include/tinyalsa/mixer.h
@@ -94,6 +94,8 @@ int mixer_subscribe_events(struct mixer *mixer, int subscribe);
 
 int mixer_wait_event(struct mixer *mixer, int timeout);
 
+int mixer_read_event(struct mixer *mixer, struct snd_ctl_event *ev);
+
 unsigned int mixer_ctl_get_id(const struct mixer_ctl *ctl);
 
 const char *mixer_ctl_get_name(const struct mixer_ctl *ctl);
-- 
1.9.1

